### DISCUSSION TITLE
Maven repository release list caching

### DISCUSSION THREAD

**Discussion by bh-tt** [renovatebot/renovate#32882] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab, 38.142.7

### Please tell us more about your question or problem

We are using renovate for gradle updates, and have implemented a basic webhook (renovate adds a footer to the issue with a checkbox, if that is checked we trigger a renovate run). Developers use this checkbox to e.g. trigger an update to a specific repository after pushing a new version of a library dependency. However, we are running into a hardcoded cachettl of 30 minutes for maven lookups: https://github.com/renovatebot/renovate/blob/1ea101c9b28bd4a0bf7f20d8d303c326f749637b/lib/modules/datasource/maven/index.ts#L110. Developers will trigger a run, renovate runs, unchecks the checkbox and does not create any branches, as the version list is still cached. 

The only alternative is disabling caching completely, but that results in a non-negligible performance impact to the renovate job. Is it possible to add a similar option as for the docker datasource? There we can set `RENOVATE_CACHE_TTL_OVERRIDE= {"datasource-docker-tags":0}`. This caches information about each release for a longer period but disables the cache for the list of docker tags.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Try https://docs.renovatebot.com/self-hosted-configuration/#cachettloverride
